setup/Microsoft.VisualStudio.NuGet.BuildTools.swr

```diff
@@ -1,7 +1,7 @@
 use vs
 
 package name=Microsoft.VisualStudio.NuGet.BuildTools
-        version=15.0.40800.$(BuildNumber)
+        version=16.0.$(MajorNuGetVersion)0$(MinorNuGetVersion)0$(PatchNuGetVersion).$(BuildNumber)
```

Comment:
```suggestion
        version=$(MajorNuGetVersion).$(MinorNuGetVersion).$(PatchNuGetVersion).$(BuildNumber)
```

Reply:
We shouldn't be using the dots there, so that suggestion won't quite work. 

Also I'm not sure if swr can do msbuild that way, but I'll try.